bot/cogs/snakes.py

```diff
@@ -40,8 +65,84 @@ def __init__(self, bot: AutoShardedBot):
         :param ctx: Context object passed from discord.py
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         """
+        # Everything with snek_list should be cached >(PC)
+        # SELF.BOT.SNEK_LIST OMG OMG OMG >(PC)
+        # Since, on restart, the bot will forget this, it will be re-cached every time. Problem Solved in theory. >(PC)
+        possible_names = 'https://en.wikipedia.org/w/api.php?action=query&titles=List_of_snakes_by_common_name' \
+                         '&prop=extracts&exlimit=1&explaintext&format=json&formatversion=2'
+
+        async with ClientSession() as session:
+            async with session.get(possible_names) as all_sneks:
+                resp = str(await all_sneks.read(), 'utf-8')
+
+        # can we find a better way to do this? Doesn't seem too reliable, even though MW won't change their api. >(PC)
+        snek_list = resp[409:].lower().split('\\n')
+
+        # if name is None, choose a random snake. Need to clean up snek_list. >(PC)
+        if name is None:
+            name = choice(snek_list)
+
+        # stops the command if the snek is not on the list >(PC)
+        elif name.lower() not in snek_list:
+            await ctx.send('This is not a valid snake. Please request one that exists.\n'
+                           'You can find a list of existing snakes here: ')
+            return
+
+        # accounting for the spaces in the names of some snakes. Allows for parsing of spaced names. >(PC)
+        if name.split(' '):
+            name = '%20'.join(name.split(' '))
+
+        # leaving off here for the evening. Building the embed is easy. Pulling the information is hard. /s >(PC)
+        # snek_dict = await self.get_snek(name)
 
     # Any additional commands can be placed here. Be creative, but keep it to a reasonable amount!
+    @command()
+    async def play(self, ctx: Context, order):
+        """
+        DiscordPlaysSnek
+
+        Move the snek around the field and collect food.
+
+        Valid use: `bot.play {direction}`
+
+        With 'left', 'right', 'up' and 'down' as valid directions.
+        """
+
+        # Maybe one game at a given time, and maybe editing the original message instead of constantly posting
+        # new ones? Maybe we could also ask nicely for DMs to be allowed for this if they aren't. >(PC)
+        if order in self.movement_command.keys():
```

Comment:
I may be wrong, but it would appear that only 1 person can play at once, you should save the message author ID to a dict with their last time? I havn't played this game, so I may be misinterpreting it.